back/src/main/resources/properties/thymeleaf.yml

```
@@ -0,0 +1,9 @@
+# local 환경
+spring:
+  config.activate.on-profile: local
+  thymeleaf:
+    prefix: classpath:/mail_template/
+    suffix: .html
+    mode: HTML
+    encoding: UTF-8
+    cache: false
```

YAMLlint (1.37.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)